<launch>
  <node pkg="tio_control" type="tio_control_node" name="tio_control_node" output="screen" respawn="false" args="/home/li/log/tio_control/ 7">
      <param name="device_id" type="int" value="3"/>
      <param name="color_weight" type="int" value="5"/>
      <param name="motor_timeout" type="int" value="60"/>
      <param name="collect_timeout" type="int" value="120"/>
      <param name="mapping_color_timeout" type="int" value="300"/>
      <param name="finish_topic" type="str" value="/ydmsg/platform/keep"/>
      <param name="mapping_file" value="$(find tio_control)/launch/mapping.json"/>
      <param name="coloring_finish_topic" type="str" value="/ydmsg/platform/keep"/>
      <param name="coloring_start_topic" type="str" value="/tio/coloring/params"/>
      <param name="coloring_timeout" type="int" value="300"/>
  </node>
</launch>